src/View/mainpage/ImageFusionView.py

```diff
@@ -0,0 +1,867 @@
+from PySide6 import QtWidgets, QtCore, QtGui
+from skimage import measure
+
+from src.Model.Isodose import get_dose_grid
+from src.Model.PatientDictContainer import PatientDictContainer
+from src.Model.ROI import get_contour_pixel
+from src.Model.ImageFusion import get_fused_pixmap, generate_comparison_colormix
+
+from src.Controller.PathHandler import resource_path
+
+import matplotlib.pyplot as plt
+
+import SimpleITK as sitk
+import numpy as np
+
+class ImageFusionView(QtWidgets.QWidget):
+    def __init__(self, roi_color=None, iso_color=None, format_metadata=False, slice_view="axial", color=False):
+        QtWidgets.QWidget.__init__(self)
+        self.patient_dict_container = PatientDictContainer()
+        self.iso_color = iso_color
+        self.zoom = 1
+        self.current_slice_number = None
+        self.slice_view = slice_view
+        '''
+        if self.slice_view == "axial":
```

Comment:
This class has a lot of similarities with DicomAxialView, can you make it an inherited class of DicomAxialView (or DicomView) so that it does not contain repeated code?

Reply:
Good pickup! We'll update to make ImageFusionView inherit from DicomView